### PR TITLE
[cpr/crc32c] Update version

### PR DESCRIPTION
**Pull request by JackBoosY**
Continue PR #13558.
Special changes:
- crc32: set it always build as a static library instead of set CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS to ON.

Related: #13536.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/devil/portfile.cmake [hunk 1]
```diff
@@ -12,57 +10,26 @@ vcpkg_from_github(
...
+    lcms    IL_NO_LCMS
+)
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}/DevIL
     PREFER_NINJA
     DISABLE_PARALLEL_CONFIGURE
     OPTIONS
+         ${FEATURE_OPTIONS}
         -DIL_NO_PNG=${IL_NO_PNG}
```
Comment (ras0219-msft) [inline ports/devil/portfile.cmake:33]: These options are now handled by `vcpkg_check_features()` and should be removed.

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

File: ports/devil/portfile.cmake (hunk 1 above)
Comment (JackBoosY) [inline ports/devil/portfile.cmake:33]:
```suggestion
```

**Comment by JackBoosY** [comment]
Blocked by #13722.

**Comment by JackBoosY** [comment]
blocked by #13765.

File: ports/devil/portfile.cmake [hunk 2]
```diff
@@ -12,58 +10,26 @@ vcpkg_from_github(
...
+)
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}/DevIL
     PREFER_NINJA
     DISABLE_PARALLEL_CONFIGURE
     OPTIONS
-        -DIL_NO_PNG=${IL_NO_PNG}
+         ${FEATURE_OPTIONS}
         -DIL_NO_TIF=${IL_NO_TIF}
```
Comment (ras0219-msft) [inline ports/devil/portfile.cmake:33]: These features (plural) are handled by `FEATURE_OPTIONS` and need to be removed:

- `IL_NO_TIF`
- `IL_NO_JPG`
- `IL_NO_EXR`
- `IL_NO_JP2`
- `IL_NO_LCMS`

These features need to always be set to `1`:
- `IL_NO_MNG`

These features need to always be set to `0`:
- `IL_USE_DXTC_NVIDIA`
- `IL_USE_DXTC_SQUISH`

Please review the entire list of options to `vcpkg_configure_cmake()` and ensure they are all set correctly.

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**